apps/docs/src/app/core/component-docs/breadcrumb/examples/breadcrumb-routerLink-example.component.html

```diff
@@ -1,12 +1,12 @@
 <fd-breadcrumb>
     <fd-breadcrumb-item>
-        <a fd-breadcrumb-link routerLink="../../avatar">Breadcrumb Level 1</a>
+        <a fd-link routerLink="../../avatar"> Breadcrumb Level <span style="font-weight: bold">1</span> </a>
```

Comment:
Why bold here?

Reply:
Just to showcase that when it transforms to the menu, html is saved, previously it was not, it would just take inner text and that's it